Lib/xml/etree/ElementTree.py

```diff
@@ -979,7 +979,7 @@ def _serialize_html(write, elem, qnames, namespaces, **kwargs):
                             k,
                             _escape_attrib(v)
                             ))
-                for k, v in sorted(items):  # lexical order
+                for k, v in items:
```

Comment:
This change has already been committed: commit 3b05ad7be09af1d4510eb698b0a70d36387f296e, PR #10190.

Reply:
You are right, I did not realize that @serhiy-storchaka have made that changes some minutes before this PR.